src/ModelTypes.h

```diff
@@ -18,14 +18,14 @@ struct msTriangle
 struct msMesh
 {
 	RString			sName;
-	char			nMaterialIndex;
+	signed char			nMaterialIndex;
```

Comment:
Let's add a comment saying we're using a signed char specifically to support ARM based architectures like the Raspberry Pi

Reply:
This but also unless it's actually a character we should use int8_t if we want an 8 bit type (strictly 'char' doesn't have to be 8 bit in c++).

I should be able to test these tomorrow or soonish, sounds promising though.

Reply:
Yeah I think I'd prefer that too actually